### DISCUSSION TITLE
[10장] 기관명의 네이밍

### DISCUSSION THREAD

**Discussion by firewoody237** [cowaine/is-my-code-that-strange#89] [Chapter 09-10 (악마, 이름 설계)] [answered]
외부와 통신하는 서비스들을 만들떄도 많은데요, 카카오, 배민 이런곳은 이름이 짧아서 괜찮지만..
가끔 기관 같은경우는 이름이 길 때가 있습니다.

예를들면 한국공항공사 같은경우
`KoreaAirportsCorporation`인데요, 줄여서 `KAC`라고도 쓸 수 있습니다.

저는 왠만해선 줄임어를 쓰지 않으려고 노력하는데요, 이런 경우 딜레마에 빠집니다..

예를들면
`KoreaAirportsCorporationSalesSearchDto`
`KoreaAirportsCorporationWarehouseStockCreateService`
처럼 너무 쓸데 없이 길어지는데요,
주석으로 기술하기엔, 모든 부분에 `KAC는 KoreaAirportsCorporation의 줄임말입니다.`라고 쓰기도 애매하고..

이런경우 줄임말을 쓰시나요?

**Comment by 0sunset0** [comment]
KAC는 검색하면 한국공항공사라고 나오는데 이정도는 축약하고 문서화를 잘 하면 괜찮지 않을까요? 
고유 명사 축약어가 사회에서 쓰이고 있다면 축약해도 의미가 사라지지 않는다고 생각합니다!

**Comment by Irisation23** [comment, marked as answer]
축약어가 많아진다면 회사에서 사용하는 Wiki 등을 이용해 축약어 모음집을 하나 만들 것 같습니다!

**Comment by corock** [comment]
`KoreaAirportsCorporation` 을 써도 되고, `KAC` 로 줄임말을 사용해도 되지만 한 프로젝트에서 두 가지를 혼용하는 것은 안 좋다고 생각합니다.
